docs/app/pages/components/components-sections/input-controls/input-dropdown/input-dropdown.component.html

```diff
@@ -83,7 +83,7 @@
         If <code>true</code>, the filter field is not displayed.
     </tr>
     <tr uxd-api-property name="maxHeight" type="number">
```

Comment:
Should `type` now be updated here to be `string | number`?

Reply:
Done.